requirements.txt

```diff
@@ -32,3 +32,12 @@ snowballstemmer==1.2.0
 traitlets==4.0.0
 websocket-client==0.34.0
 wsgiref==0.1.2
+
```

Comment:
move this to setup.py
@nsoblath is there a graceful way on getting the specific git commit installed? submodule maybe?